dmoj/executors/shell_executor.py

```diff
@@ -21,10 +21,10 @@ def get_fs(self):
     def get_allowed_syscalls(self):
         return super().get_allowed_syscalls() + ['fork', 'waitpid', 'wait4']
 
-    def get_security(self, launch_kwargs=None):
+    def get_security(self, launch_kwargs=None, extra_fs=None):
```

Comment:
Not that this is a direct result of this PR, but maybe this should be converted to `**kwargs` instead?

Reply:
We can do that later.